README.md

```diff
@@ -21,35 +21,36 @@ Given that different groups of genes both contribute to these traits of interest
+| [4_acc_acc_analysis](4_acc_acc_analysis) | This folder contains analysis notebooks to examine accessory-accessory gene modules.|
+| [5_core_core_analysis](5_core_core_analysis) | This folder contains analysis notebooks to examine the stability of core genes across strains.|
+| [6_core_acc_analysis](6_core_acc_analysis) | This folder contains analysis notebooks to examine the relationship between core genes and accessory genes.|
+| [scripts](scripts) | This folder contains supporting functions that other notebooks in this repository will use.|
 | [data](data) | This folder contains metadata used for different analyses.|
-| [explore_data](explore_data) | This folder contains analysis notebooks to visualize the expression data to get a sense for the variation contained.|
-| [processing](processing) | This folder contains analysis notebooks to determine what threshold to use to partition the gene expression data into PAO1 and PA14 compendia.|
 
 
 ## Usage
-If you want to run notebooks from core-accessory-interactome, you need to follow these steps to create a conda environment and download the necessary data.
+*Operating Systems:* Mac OS, Linux (Note: bioconda libraries not available in Windows)
 
-1. In the terminal execute the following steps to setup the conda environment:
-```bash
-# conda version 4.6.12
-conda env create -f environment.yml
-conda activate core_acc
-pip install -e .
-```
-
-2. Update paths in `core_acc_modules/paths.py`
+In order to run this simulation on your own gene expression data the following steps should be performed:
 
-3. Download reference transcriptomes from Pseudomonas.com and store in appropriate path defined in `core_acc_modules/paths.py`
-
-4. In the terminal launch jupyter notebook within your environment:
+First you need to set up your local repository:
+1. Download and install [github's large file tracker](https://git-lfs.github.com/).
+2. Install [miniconda](https://docs.conda.io/en/latest/miniconda.html)
+3. Clone the `core-accessory-interactome` repository by running the following command in the terminal:
+```
+git clone https://github.com/greenelab/core-accessory-interactome.git
+```
+Note: Git automatically detects the LFS-tracked files and clones them via http.
+4. Navigate into cloned repo by running the following command in the terminal:
+```
+cd core-accessory-interactome
+```
+5. Set up conda environment by running the following command in the terminal:
 ```bash
-jupyter notebook
+bash install.sh
 ```
-
-**WARNING: This section is a work in progress and will be updated as the project develops**
\ No newline at end of file
+6. Navigate to any of the analysis directories listed in the table above to see the code for how analyses were performed. To reproduce the results and figures of the analysis, run the analysis directories in order.
```

Comment:
```suggestion
6. Navigate to any of the analysis directories listed in the table above to see the code for how analyses were performed. To reproduce the results and figures of the paper, run the analysis directories in order.
```

Reply:
Check this change to make sure it's correct (mostly the change is just to avoid using analysis too many times and ending up confusing the reader)